src/styles/blocks/card.css

```
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  max-width: 200px;
```

```suggestion
  width: 200px;
```

card should always have 200px width